content/getting_started/from_the_query_to_the_graph.md

```diff
@@ -22,7 +22,7 @@ At each step we comment on the effect of each parameter of the query.
 
 The metric `system.disk.total` (collected by default by the [datadog-agent](/agent)) is seen from different sources.  
 
-First because this metric reported by different hosts, and also because each datadog-agent collects this metric device per device. It adds to metric `system.disk.total` the tag `device:tmpfs` when sending data associated to the disk with the same name, etc.
+First because this metric is reported by different hosts, and also because each datadog-agent collects this metric device per device. It adds to metric `system.disk.total` the tag `device:tmpfs` when sending data associated to the disk with the same name, etc.
```

Comment:
This is ok, but let's also remove the first device in "each datadog-agent collects this metric device per device" so it's just "each datadog-agent collects this metric per device"